packages/desktop/components/Settings/RecordingSettings.tsx

```diff
@@ -73,6 +83,36 @@ const recStates: Record<RecStatus | 'EngineNotStarted', { icon: JSX.Element; mes
   },
 };
 
+function PreviewVideoWindow({ key }: { key: string }) {
```

Comment:
hmm.. would it be possible for window resizing to affect the rect?

Reply:
atm the div is statically sized with css